pkg/sql/ttl/ttlschedule/ttlschedule.go

```diff
@@ -217,12 +217,18 @@ func createRowLevelTTLJob(
 	ctx context.Context,
 	createdByInfo *jobs.CreatedByInfo,
 	txn *kv.Txn,
-	ie sqlutil.InternalExecutor,
+	descsCol *descs.Collection,
 	jobRegistry *jobs.Registry,
 	ttlDetails catpb.ScheduledRowLevelTTLArgs,
 ) (jobspb.JobID, error) {
+	f := tree.NewFmtCtx(tree.FmtSimple)
+	tn, err := descs.GetTableNameByID(ctx, txn, descsCol, ttlDetails.TableID)
+	if err != nil {
+		return 0, err
+	}
+	f.FormatNode(tn)
 	record := jobs.Record{
-		Description: "ttl",
+		Description: fmt.Sprintf("ttl for %s", f.CloseAndGetString()),
```

Comment:
hmm are we allowed to include identifiers in this description? it could cause it to get logged and bypass the log redaction for identifiers, right?

Reply:
I assume if anything logs the description they’d wrap the whole thing in redaction markers ?

Reply:
sounds right

Reply:
oh, though i noticed that this is kinda duplicating the logic of just calling `tn.String()` (or `tree.AsString(tn)`). just a nit, not a big deal